### ISSUE TITLE
Using an instanceId

### ISSUE THREAD

**Issue by 3cooper** [zalmoxisus/redux-devtools-extension#144]
I'm using version 2.0.1.  I calling the store like this:

      oStore = Redux.createStore(
        oReducer,
        oInitialStore,
        document.cookie.indexOf('redux-debug') > -1 &&  window.devToolsExtension ? window.devToolsExtension({ name: 'MyName', instanceId: Math.floor(Date.now() / 1000).toString()}) : f => f
      );

I have this again for another component with MyName2.  MyName component loads first and I can see its information devtools.  Once I load a second component with MyName2, the data in the inspector for MyName is replaced by the data for MyName2.   I can select between both stores but the data is the same.  Am I doing something wrong?

**Comment by zalmoxisus** [comment]
@3cooper, by default it autoselect the instance. If you want to see the information for specific instance, you should select it explicitly
<img width="538" alt="rext" src="https://cloud.githubusercontent.com/assets/7957859/16221994/785c2e26-379e-11e6-830a-ac93313a24a9.png">
Selecting the instance is not present in Chrome DevTools panel for now, so you should open it in a window (click the first button on the bottom).

BTW, `instanceId` by default already should have a random value, so no need to specify it there.

**Comment by 3cooper** [comment]
I did open devtools in a window.  I have also now removed the instanceId.   I see both my instances in the menu but when i switch between them they are the same.  When it loads I see my first instance and it has many actions logged.  Then I click a button that loads another component onto the page.  When that new component loads I can see both instances in the devtools window but when I switch between them they contain the same information which is the INIT action for the most recently added store.  It seems to have lost the information that was previously in there for the first store and overwritten it with the new one.
